tests/framework/hybridModel/test_hybrid_model.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<Simulation verbosity="debug">
+<Simulation verbosity="silent">
```

Comment:
Unless there is a really good reason, quiet (which still allows warnings and errors) should be used instead of silent.